interaction-manager/src/utils/global-object.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Global object class
+ */
+class IM {
+    // eslint-disable-next-line require-jsdoc
+    constructor() {
+        this.actions = [];
+        this.keyboardFunctions = [];
+        this.gamepadFunctions = [];
+    }
+
+    /**
+     * Initialize global object
+     */
+    init() {
+        if (!window._IM) window._IM = new IM();

```

Comment:
What if I have defined `_IM` and it is not an instance of `IM`? :D I know the chance is minimalistic but .. :D